src/renderer/components/+workloads-pods/pod-details-list.module.scss

```diff
@@ -0,0 +1,56 @@
+.PodDetailsList {
+  @mixin status-colors {
+    @each $status, $color in $pod-status-color-list {
+      &:global(.#{$status}) {
+        color: $color;
+      }
+    }
+  }
+
+  .table {
+    &:global(.virtual) {
+      height: 500px;
+    }
+
+    :global(.TableCell).warning {
+      @include table-cell-warning;
+    }
+
+    :global(.TableCell).status {
+      @include status-colors;
+    }
```

Comment:
Probably better to re-assign new module classes to each of `:global()` counterpart within `Table`. 
I hope all of inner Table components support extra `className` prop, so you can use module-generated class names benefit already no tightening itself with global implementation.